src/libslic3r/PresetBundle.cpp

```diff
@@ -43,11 +43,17 @@ static std::vector<std::string> s_project_options {
     "flush_multiplier",
 };
 
-//BBS: add BBL as default
-const char *PresetBundle::BBL_BUNDLE = "Custom";
-const char *PresetBundle::BBL_DEFAULT_PRINTER_MODEL = "MyKlipper 0.4 nozzle";
-const char *PresetBundle::BBL_DEFAULT_PRINTER_VARIANT = "0.4";
-const char *PresetBundle::BBL_DEFAULT_FILAMENT = "My Generic PLA";
+////BBS: add BBL as default
+//const char *PresetBundle::BBL_BUNDLE = "Custom";
+//const char *PresetBundle::BBL_DEFAULT_PRINTER_MODEL = "MyKlipper 0.4 nozzle";
+//const char *PresetBundle::BBL_DEFAULT_PRINTER_VARIANT = "0.4";
```

Comment:
We don't rename variables to keep minimum changes.

Reply:
Ok. I will delete them